package-lock.json

```diff
@@ -22521,14 +22521,6 @@
         "lodash": "^4.17.21"
       }
     },
-    "sort-keys": {
```

Comment:
Probably cause of a previous merge, `sort-keys` appeared twice in the package-lock file.